src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/displayDescriptionData_List.cs

```diff
@@ -379,22 +379,23 @@ private ListEntryBuilder AddItem(string value, string label, DisplayEntryValueTy
             {
                 DisplayEntry = new DisplayEntry(value, kind),
                 Label = label,
-                FormatString = format
+                FormatString = format,
+                ItemSelectionCondition = itemSelectionContition,
             });
 
             return this;
         }
 
         /// <summary></summary>
-        public ListEntryBuilder AddItemScriptBlock(string scriptBlock, string label = null, string format = null)
+        public ListEntryBuilder AddItemScriptBlock(string scriptBlock, string label = null, string format = null, DisplayEntry itemSelectionContition = null )
```

Comment:
Is the change binary backward compatible? We shouldn't break public API.

Reply:
The change is not binary compatible.

That said, it is unlikely anyone uses this api yet, people still rely on a `ps1xml` file because there isn't a good way for a module author to use this api in a useful way yet.